lib/src/clojure_lsp/feature/completion.clj

```diff
@@ -217,36 +217,38 @@
                                 set)]
       (concat
         (when (simple-ident? cursor-value)
-          (->> aliases
-               (filterv (fn [element]
-                          (or
-                            (matches-fn (:alias element))
-                            (matches-fn (:to element)))))
-               (mapv
-                 (fn [element]
-                   (let [require-edit (some-> cursor-loc
-                                              (f.add-missing-libspec/add-known-alias (symbol (str (:alias element)))
-                                                                                     (symbol (str (:to element)))
-                                                                                     db)
-                                              r.transform/result)]
-                     (cond-> (element->completion-item element nil :required-alias)
-                       (seq require-edit) (assoc :additional-text-edits (mapv #(update % :range shared/->range) require-edit))))))))
-        (->> (:analysis db)
-             (mapcat val)
-             (keep
-               #(when (and (identical? :var-definitions (:bucket %))
-                           (not (:private %))
-                           (contains? alias-namespaces (:ns %))
-                           (or (simple-ident? cursor-value) (matches-fn (:name %))))
-                  [(:ns %) (element->completion-item % cursor-alias :unrequired-alias)]))
-             set
-             (map
-               (fn [[element-ns completion-item]]
-                 (let [require-edit (some-> cursor-loc
-                                            (f.add-missing-libspec/add-known-alias (symbol cursor-alias) element-ns db)
-                                            r.transform/result)]
-                   (cond-> completion-item
-                     (seq require-edit) (assoc :additional-text-edits (mapv #(update % :range shared/->range) require-edit)))))))))))
```

Comment:
The logic has changed here. It used to be
```clojure
(->> elements
     (filter var-def?)
     (map (juxt ns completion-item))
     distinct
     (map with-require-edits))
```

Now it's
```clojure
(->> elements
     (filter var-def?)
     (map (juxt ns completion-item))
     (map with-require-edits)
     distinct)
```
So, two questions. 1. is it still correct? And 2. is it actually faster? Usually, moving a `distinct` later hurts performance, because you have to do extra work that will be thrown away.

Reply:
Not I got the code snippet, but I only removed the `set` since we are `into #{}` now

Reply:
Yeah, looks right 👍